### PR TITLE
[Console] Document the silent verbosity level

### LINKED ISSUE #20243: [Console] Add silent verbosity suppressing all output, including errors
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#53632
| PR author(s) | @wouterj
| Merged in    | 7.2

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 7.2 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by javiereguiluz**
Fixes #20243.

### CODE REVIEW

File: components/console/usage.rst [hunk 1]
```diff
@@ -65,9 +65,17 @@ You can suppress output with:
...
 .. code-block:: terminal
 
+    # suppresses all output except errors
     $ php application.php list --quiet
     $ php application.php list -q
 
+    # suppresses all output, including errors
+    $ php application.php list --silent
+
+.. vesionadded:: 7.2
```
Comment (xabbuh) [inline components/console/usage.rst]:
```suggestion
.. versionadded:: 7.2
```

File: console/input.rst [hunk 2]
```diff
@@ -446,12 +446,17 @@ The Console component adds some predefined options to all commands:
...
   ``3``, or you can use respective shortcuts ``-v``, ``-vv`` and ``-vvv``)
-* ``--quiet``: disables output and interaction
+* ``--quiet``: disables output and interaction (errors are still displayed)
+* ``--silent``: disables all output and interaction, including errors
 * ``--no-interaction``: disables interaction
 * ``--version``: outputs the version number of the console application
 * ``--help``: displays the command help
 * ``--ansi|--no-ansi``: whether to force of disable coloring the output
 
+.. vesionadded:: 7.2
```
Comment (xabbuh) [inline console/input.rst]:
```suggestion
.. versionadded:: 7.2
```

File: console/verbosity.rst [hunk 3]
```diff
@@ -30,13 +33,18 @@ have more precedence over the value of ``SHELL_VERBOSITY``):
...
 =====================  =========================  ===========================================
+``--silent``           ``-2``                     ``OutputInterface::VERBOSITY_SILENT``
 ``-q`` or ``--quiet``  ``-1``                     ``OutputInterface::VERBOSITY_QUIET``
 (none)                 ``0``                      ``OutputInterface::VERBOSITY_NORMAL``
 ``-v``                 ``1``                      ``OutputInterface::VERBOSITY_VERBOSE``
 ``-vv``                ``2``                      ``OutputInterface::VERBOSITY_VERY_VERBOSE``
 ``-vvv``               ``3``                      ``OutputInterface::VERBOSITY_DEBUG``
 =====================  =========================  ===========================================
 
+.. vesionadded:: 7.2
```
Comment (xabbuh) [inline console/verbosity.rst]:
```suggestion
.. versionadded:: 7.2
```

File: components/console/usage.rst (hunk 1 above)
Comment (javiereguiluz) [reply components/console/usage.rst]: Fixed. Thanks!

**Comment by wouterj** [comment]
You're fast Javier. thank you!

I was thinking we might also want to document `$output->isSilent()` somewhere. E.g. I can think of a pattern where you write some information to a logger when a command is run in silent mode (if the command is ran both manually and programmatically). If we do it, it's good to mention that errors will always be logged to the logger by the Symfony framework integration.

**Comment by javiereguiluz** [comment]
Wouter, you are right! I added some more content. Let me know if it's OK. Thanks.

**Comment by javiereguiluz** [comment]
Let's merge this one and we can create other PRs if we want to change or improve things. Thanks!

**Pull request merged**